src/BenchmarkDotNet/Attributes/Obsolete/DryCoreJobAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+using System;
+
+namespace BenchmarkDotNet.Attributes
+{
+    [Obsolete("Please use DryJobAttribute instead. Use the ctor that requires RuntimeMoniker argument.", false)]
```

Comment:
Instead of "Use the ctor that requires RuntimeMoniker argument", it will be better to write a code example that can be copy-pasted.